app/controllers/cogy/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Cogy
   class ApplicationController < ActionController::Base
     # @todo https://github.com/skroutz/cogy/issues/43
-    skip_before_action :verify_authenticity_token
+    protect_from_forgery unless: -> { request.format.json? }
```

Comment:
We don't want to enable CSRF protection at all: enabling it even on non-JSON requests will cause requests from the [cogy-bundle](https://github.com/skroutz/cogy-bundle) to be rejected.